src/views/Forbidden.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div>
+    <empty-state :icon="'ban'" :heading="$t('403.title')" :body="$t('403.body')">
+      <router-link class="btn btn--ter" :to="{ name: 'Home' }">
+        {{ $t('403.linkText') }}
+      </router-link>
+    </empty-state>
+  </div>
+</template>
+
+<script>
+import i18n from '@/locale/i18n';
+import EmptyState from '@/components/EmptyState.vue';
+
+export default {
+  name: 'Forbidden',
+  components: {
+    EmptyState,
+  },
+
+  metaInfo() {
+    return {
+      title: `${i18n.t('403.title')} | ${i18n.t('general.project')}`,
+    };
+  },
```

Comment:
Ser ut til å være et mer generelt problem, men denne tittelen blir stuck selv om man for eksempel går tilbake til forsiden. :/

Reply:
Hm, se det. Virker ikke helt korrekt å redirecte til en ny path heller, som i dag for 404? Men får [se på det](https://github.com/vuejs/vue-router/issues/977) + bruken av [vue-meta](https://vue-meta.nuxtjs.org/guide/metainfo.html) separat kanskje.